README.md

```diff
@@ -1,2 +1,127 @@
+**NOTE:** this is a draft README for not fully implemented yet feature.
+
 # edx-calendar-tab
 Shared calendar for Open edX Courseware
+
+
+Django application that allows course staff enable new "Calendar" tab from CMS (studio).
+From LMS side appears new "Calendar" tab with google calendar associated with current course.
+New calendar events may be created by any enrolled student.
+Calendar may be shared by stuff personal with provided google accounts (with update permission).
+
+1) INSTALLATION
+
+1.1) Install Package
+
+installing manually for evaluation and testing:
+
+    sudo su - edxapp -s /bin/bash
+    . edxapp_env
+    pip install -e git+https://github.com/raccoongang/edx-calendar-tab.git@wowkalucky/event_permissions#egg=edx-calendar-tab
+
+
+
+1.2) Production installation
+
+#TBD
+
+
+2) CONFIGURATION
+
+2.1) Configure edx-platform
+
+Add "edx-calendar-tab" to installed Django apps
+
+In "/edx/app/edxapp/lms.env.json" add
+
+    "ADDL_INSTALLED_APPS": ["calendar_tab"],
+
+In "/edx/app/edxapp/edx-platform/lms/envs/common.py" add
+
+    MAKO_TEMPLATES['main']: [
+        ...
+        '/edx/app/edxapp/venvs/edxapp/src/edx-calendar-tab/calendar_tab/templates',
+    ]
+
+    PIPELINE_CSS = {
+        ...
+        'style-calendar-tab': {
+            'source_filenames': [
+                '/edx/app/edxapp/venvs/edxapp/src/edx-calendar-tab/calendar_tab/static/calendar_tab/css/vendor/scheduler/dhtmlxscheduler.css',
+            ],
+            'output_filename': 'css/calendar-tab.css',
+        }
+    }
+
+    PIPELINE_JS = {
+        ...
+        'calendar_tab': {
+            'source_filenames': [
+                '/edx/app/edxapp/venvs/edxapp/src/edx-calendar-tab/calendar_tab/static/calendar_tab/js/calendar-tab.js',
+            ],
+            'output_filename': 'js/calendar_tab.js',
+        },
+        'calendar_tab_vendor': {
+            'source_filenames': [
+                '/edx/app/edxapp/venvs/edxapp/src/edx-calendar-tab/calendar_tab/static/calendar_tab/js/vendor/scheduler/_dhtmlxscheduler.js',
+                '/edx/app/edxapp/venvs/edxapp/src/edx-calendar-tab/calendar_tab/static/calendar_tab/js/vendor/scheduler/dhtmlxscheduler_readonly.js',
+            ],
+            'output_filename': 'js/calendar_tab_vendor.js',
+        },
+    }
+
+In "/edx/app/edxapp/edx-platform/lms/urls.py" add __before__ static_tab urls:
```

Comment:
I see it, but not sure I know the way to avoid extending 'INSTALLED_APPS' and 'urlpatterns' during our app installation...